orttraining/orttraining/python/experimental/orttrainer.py

```diff
@@ -451,12 +451,19 @@ def forward(self, *inputs):
         # Do an inference to grab output types
         model.eval()
         with torch.no_grad():
-            # Deepcopy inputs, since input values may change after model run.
-            sample_inputs_copy = copy.deepcopy(sample_inputs)
-            # Deepcopy model, in case model is stateful and changes after model run.
-            model_copy = copy.deepcopy(model)
+            try:
+                # Deepcopy inputs, since input values may change after model run.
+                sample_inputs_copy = copy.deepcopy(sample_inputs)
+                # Deepcopy model, in case model is stateful and changes after model run.
+                model_copy = copy.deepcopy(model)
+            except:
```

Comment:
Please specify the exception you want to catch, or at lease use `except Exception:`, so things like keyboard interrupt and system exit will not be caught.

Reply:
It depends on the model. Can be TypeError, pickle.PickingError, cpickle.PicklingError, etc
Each python module can wrap the original PicklingError and raise a different exception

regardless, will change it to except Exception